go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -493,9 +503,11 @@ func addDefaultConntrackRules(nodeName, gwBridge, gwIntf string, stopChan chan s
 	}
 	nFlows++
 
-	// table 1, all other connections do normal processing
+	// table 1, unknown L3/L4 traffic go to host
+	// we have already CT'ed this IP traffic, so we know it did not belong to OVN backed k8s service
+	// and we have no match so we know it must not be for OVN
 	_, stderr, err = util.RunOVSOfctl("add-flow", gwBridge,
-		fmt.Sprintf("cookie=%s, priority=0, table=1, actions=output:FLOOD", defaultOpenFlowCookie))
+		fmt.Sprintf("cookie=%s, priority=0, table=1, actions=LOCAL", defaultOpenFlowCookie))
```

Comment:
Due to some CT bug with icmpv6 for ND we need to still flood icmpv6 (i think that's why the ipv6 job is having some failures). @Billy99 can you link me a reference to that bug again please?

Reply:
@trozet 
https://bugzilla.kernel.org/show_bug.cgi?id=11797